deploy/manifests/config-map.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+data:
+  config.yaml: |-
+    "resourceRules":
+      "cpu":
+        "containerLabel": "container"
+        "containerQuery": |
+          sum by (<<.GroupBy>>) (
+            irate (
+                container_cpu_usage_seconds_total{<<.LabelMatchers>>,container!="",pod!=""}[4m]
+            )
+          )
+        "nodeQuery": |
```

Comment:
For node queries, I would prefer if we stick to kubernetes metrics as kubernetes should be the only dependency of the default installation of prometheus-adapter. So instead of using metrics from node-exporter and windows-exporter, we could reuse the container queries but with `id='/'` as an additional label selector to get the resource usage of the node.

In the future, we could make the different setup configurable via jsonnet, but for now, let's just stick to the default use-case.